mppsolar/version.py

```diff
@@ -1,5 +1 @@
-import configparser
-
-version = configparser.ConfigParser()
-version.read("pyproject.toml")
-__version__ = version["tool.poetry"]["version"].replace('"', "")
+__version__ = "0.15.30"
```

Comment:
can you revert / change this to the configparser version

Reply:
It doesn't compile when I do.

Reply:
how so? what fails?

Reply:
mpp-solar-mppsolar-1      | Traceback (most recent call last):
mpp-solar-mppsolar-1      |   File "/usr/local/bin/powermon", line 5, in <module>
mpp-solar-mppsolar-1      |     from powermon import main
mpp-solar-mppsolar-1      |   File "/mpp-solar/powermon/__init__.py", line 10, in <module>
mpp-solar-mppsolar-1      |     from mppsolar.protocols import get_protocol
mpp-solar-mppsolar-1      |   File "/mpp-solar/mppsolar/__init__.py", line 5, in <module>
mpp-solar-mppsolar-1      |     from mppsolar.version import __version__  # noqa: F401
mpp-solar-mppsolar-1      |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
mpp-solar-mppsolar-1      |   File "/mpp-solar/mppsolar/version.py", line 5, in <module>
mpp-solar-mppsolar-1      |     __version__ = version["tool.poetry"]["version"].replace('"', "")
mpp-solar-mppsolar-1      |                   ~~~~~~~^^^^^^^^^^^^^^^
mpp-solar-mppsolar-1      |   File "/usr/local/lib/python3.11/configparser.py", line 979, in __getitem__
mpp-solar-mppsolar-1      |     raise KeyError(key)
mpp-solar-mppsolar-1      | KeyError: 'tool.poetry'
mpp-solar-mppsolar-1 exited with code 1

Reply:
bummer, so fails in the installed version
i really hated updating in 2 places